src/v2/Apps/Collect/Routes/Collection/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx

```diff
@@ -124,16 +114,21 @@ export const FeaturedCollectionEntity: React.FC<FeaturedCollectionEntityProps> =
             })}
           />
         </Flex>
-        <Serif size="4" mt={1} maxWidth={["246px", "100%"]}>
+        <Text variant="subtitle" mt={1} maxWidth={["246px", "100%"]}>
           <Truncator maxLineCount={1}>{title}</Truncator>
-        </Serif>
+        </Text>
         {price_guidance && (
-          <Sans size="2" color="black60">{`From $${formattedPrice}`}</Sans>
+          <Text variant="small" color="black60">
+            {`From $${formattedPrice}`}
+          </Text>
         )}
-        <ExtendedSerif size="3" mt={1}>
-          <Media lessThan="md">{getTruncatedDescription(4)}</Media>
-          <Media greaterThan="sm">{getTruncatedDescription(3)}</Media>
-        </ExtendedSerif>
+        {/* TODO: fix this truncation/overflow issue properly, rather than this overflow:hidden Box */}
+        <Box style={{ overflow: "hidden" }}>
+          <ExtendedText mt={1}>
+            <Media lessThan="md">{getTruncatedDescription(4)}</Media>
+            <Media greaterThan="sm">{getTruncatedDescription(3)}</Media>
+          </ExtendedText>
+        </Box>
```

Comment:
Oh, and one quick ugly fix for this overflow problem — 

![Screen Shot 2020-09-21 at 4 34 45 PM](https://user-images.githubusercontent.com/140521/93818460-657fff00-fc28-11ea-8ac4-b36111d84310.png)

@dzucconi you mentioned something was in flight for this?

Reply:
Oh @nicoleyeo  I was commenting on that just as you wrote! 😄

Reply:
Ah, I see; [I *think* my PR may fix this](https://github.com/artsy/palette/pull/782) — but I'll probably have to check this explicitly. That component is quite odd — I'm not sure how deep I should go with it. For this it seems like it'd make sense to just hard truncate it without `ReadMore` — if you wanted to actually read more you'd probably click on the thumbnail to go the page.

Reply:
Hmm, I haven't cracked this yet. Meanwhile @nicoleyeo has [ticketed it here](https://artsyproduct.atlassian.net/browse/FX-2289), so I'm inclined to merge this and follow up. It's an "improvement" over the status quo.